static/src/js/components/FormNavigation/FormNavigation.jsx

```diff
@@ -0,0 +1,184 @@
+ * @property {Object} schema UMM Schema.
+ * @property {Function} setFocusField A function that sets the focus field.
+ * @property {Object} uiSchema A uiSchema for the field being shown.
+ * @property {Array} visitedFields An array with a list of visited fields.
+ */
+
+/**
+ * Renders Form Navigation
+ * @param {FormNavigation} props
+ */
+const FormNavigation = ({
+  draft,
+  formSections,
+  loading,
+  onCancel,
+  onSave,
+  schema,
+  setFocusField,
+  // TODO MMT-####
+  // uiSchema,
+  visitedFields
+}) => {
+  const cleanedDraft = cloneDeep(removeEmpty(draft))
+  const { errors } = validator.validateFormData(cleanedDraft, schema)
+
+  return (
+    <>
+      <div className="mb-4">
+        <Dropdown as={ButtonGroup}>
+          <Button
+            className="text-white"
+            disabled={loading}
+            onClick={() => onSave(saveTypes.saveAndContinue)}
+            variant="success"
+          >
+            {
+              loading && (
+                <Spinner
+                  animation="border"
+                  as="span"
+                  className="me-2"
+                  role="status"
+                  size="sm"
+                />
+              )
+            }
+            <span>
+              {
+                loading
+                  ? 'Saving draft...'
+                  : 'Save & Continue'
+              }
+            </span>
+          </Button>
+
+          <Dropdown.Toggle
+            aria-label="Save Options"
+            className="text-white"
+            id="dropdown-split-basic"
+            split
+            variant="success"
+          />
+
+          <Dropdown.Menu>
+            <Dropdown.Item
+              onClick={() => onSave(saveTypes.save)}
+            >
+              <span>
+                Save
+              </span>
+            </Dropdown.Item>
+
+            <Dropdown.Item
+              onClick={() => onSave(saveTypes.saveAndContinue)}
+            >
+              <span>
+                Save &amp; Continue
+              </span>
+            </Dropdown.Item>
+
+            <Dropdown.Item
+              onClick={() => onSave(saveTypes.saveAndPublish)}
+            >
+              Save &amp; Publish
+            </Dropdown.Item>
+
+            <Dropdown.Item
+              onClick={() => onSave(saveTypes.saveAndPreview)}
+            >
+              <span>
+                Save &amp; Preview
+              </span>
+            </Dropdown.Item>
+          </Dropdown.Menu>
+        </Dropdown>
+
+        <Button
+          className="link-button ms-2"
+          onClick={onCancel}
+          type="button"
+          variant="link-secondary"
+        >
+          Cancel
+        </Button>
+      </div>
+
+      <ListGroup className="form-navigation__sections p-3 bg-light">
+        <For each={formSections}>
+          {
+            (section, index) => {
+              const { displayName } = section
+
+              // TODO MMT-####
```

Comment:
Is this something we had in the prototype?

Reply:
I was waiting on a ticket Trevor was going to write lol, need to get a real ticket number in each place that shows up